packages/client/webapp/src/utils/uncloak.ts

```diff
@@ -1,17 +1,11 @@
-let accessToken: string | null;
-
-export const getAccessToken = async () => {
-	const response = await fetch('/silent-check-sso.html');
-
-	if (response.ok) {
-		accessToken = response.headers.get('OIDC_access_token');
-	}
-};
+import { useAuthStore } from '../stores/auth';
 
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
We should remove this eslint rule instead of adding this `eslint-disable-next-line`

Reply:
I have a set of linter rules changes in my branch for the graph and scaffolding utilities (https://github.com/DARPA-ASKEM/TERArium/tree/porting-graph-scaffold) some of which have already discussed with various folks, I think let's take the linter discussion there rather than doing this exercise multiple times.